TTBot/Services/ExcelWrapper.cs

```diff
@@ -16,14 +16,13 @@ public ExcelWrapper()
             ExcelPackage.LicenseContext = LicenseContext.NonCommercial;
         }
 
-        public List<ExcelDataModel> GetExcelData(Attachment attachment)
+        public List<ExcelDriverDataModel> GetExcelDriverData(Attachment attachment)
         {
-            List<ExcelDataModel> excelDataModels = new List<ExcelDataModel>();
+            List<ExcelDriverDataModel> excelDriverDataModels = new List<ExcelDriverDataModel>();
 
             WebClient webClient = new WebClient();
             byte[] buffer = webClient.DownloadData(attachment.Url);
-            MemoryStream stream = new MemoryStream(buffer);
-
+            using MemoryStream stream = new MemoryStream(buffer);
```

Comment:
Does this compile? I'd have thought it'd need to be 

```
using (MemoryStream stream = new MemoryStream(buffer))
using (var package = new ExcelPackage(stream))
            {
```

Reply:
I had your change, but Visual Studio suggested this instead.